lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2546,24 +2546,45 @@ namespace {
       return result;
     }
 
+    enum RetainReleaseOperatonKind {notAfunction, doesntReturnVoid, invalidParameters, valid};
```

Comment:
I think most enum cases start with an upper case letter in our codebase. Also, is this how clang-format would format this?

Reply:
If this is only used within `validateForeignReferenceType`, we can potentially move it in the function to reduce the scope of this enum.